apps/pie-storybook/stories/pie-toast.stories.ts

```diff
@@ -2,35 +2,94 @@ import { html } from 'lit';
 
 /* eslint-disable import/no-duplicates */
 import '@justeattakeaway/pie-toast';
```

Comment:
Do you need both this line and the next out of interest? I think you can pull `PieToast` out from line 5? If you can then you might be able to remove the eslint-disable lines too.

Reply:
This came from another examples from other components. Honestly I don't know why we used to use this import back in the day. Removed now!